lib/sequel/plugins/uuid.rb

```diff
@@ -0,0 +1,89 @@
+# frozen-string-literal: true
+
+##
+# LOOKER CUSTOM NOTE
+# -----------------------------------------------------------------------------
+# This file has been manually ported from  a future sequel version to our
+# custom 4.2.0 version of Sequel and is, therefore, not officially supported
+# in this version.
+#
+# Modifications that have been made to support this version are:
+#   * Changed set_column_value, which is not defined in this version, to send
```

Comment:
are these comments stale?  I don't see a `set_column_value` here but the comment makes me think it should be here somewhere?

Reply:
I honestly just copy and pasted the file from the the future version and brought it back here and changed the 2 functions that don't exist in this version. I can clean up the comments to be accurate.

Reply:
cool.  Thanks!